docs/guide/getting-started.md

```diff
@@ -0,0 +1,26 @@
+# Getting started
+
+## Prerequisites
+
+- [Git](https://git-scm.com)
+- [Node.js](https://nodejs.org/en/) >= 12.13
```

Comment:
If this is hardcoded, this won't be updated 😄

Reply:
Same with the root `README.md` file.

When **Prerequisites** will evolve then we need to update it here too.